README.md

```diff
@@ -385,6 +385,14 @@ Specify whether to display ghost text.
 Default: `false`
```

Comment:
Thank you for your contribution.
Could you modify `experimental.ghost_text` type as `type: cmp.GhostTextConfig|"false"` too?

Reply:
I'd be happy to make a change! I'm learning something new here though. What does this change do and where does it go?

Reply:
All configuration options were defined here https://github.com/hrsh7th/nvim-cmp/blob/main/lua/cmp/types/cmp.lua#L93

I'd forgotten to follow it with README.md

Reply:
Like this? This is my first time working with Lua annotations 😓  
![image](https://user-images.githubusercontent.com/59481467/132378011-3aabb891-ef83-4b50-968c-11dea367ec60.png)